src/components/MovieCard/MovieCard.tsx

```diff
@@ -1,7 +1,46 @@
 import React from 'react';
 
 import './MovieCard.scss';
+// import { Movie } from '../../Types';
 
-export const MovieCard: React.FC = () => (
-  <>Put the card here</>
+interface Props {
+  title: string;
+  description: string;
+  imgUrl: string;
+  imdbUrl: string;
+}
+
+export const MovieCard: React.FC<Props> = ({
+  title, description, imgUrl, imdbUrl,
+}) => (
```

Comment:
You could have taken the `movie` and use it like below so that you won't have to feed 
`<MovieCard key={movie.imdbId} movie={movie}/>` 
one by one.

```suggestion
export const MovieCard: React.FC<Movie> = ({ movie }) => (
  const { title, description, imgUrl, imdbUrl, } = movie
```

Reply:
Actually in today's morning lesson Paul told about this method so I tried it here